pdc_client/runner.py

```diff
@@ -167,32 +167,13 @@ def run(self, args=None):
         self.client = pdc_client.PDCClient(self.args.server, page_size=self.args.page_size, ssl_verify=ssl_verify)
         try:
             self.args.func(self.args)
-        except beanbag.BeanBagException as exc:
-            self.print_error_header(exc)
-            json = None
+        except beanbag.BeanBagException as ex:
+            print("Server returned following error: [{0}] {1}".format(ex.response.status_code, ex.response.reason), file=sys.stderr)
+            print("Details:", file=sys.stderr)
             try:
-                json = exc.response.json()
-                pretty_print(json, file=sys.stderr)
-            except ValueError:
-                # Response was not JSON
-                print('Failed to parse error response.', file=sys.stderr)
-            except TypeError as e:
-                # Failed to pretty print.
-                print('Failed to correctly display error message. Please file a bug.',
-                      file=sys.stderr)
-                self.logger.info(json, exc_info=e)
```

Comment:
Fixed. Now it prints:

```
Server returned following error: [500] INTERNAL SERVER ERROR
Details:
Failed to parse error response.
```

BTW, for that particular `pdc` command, I think server should send JSON with the error. Looks like uncaught exception on server:

```
FieldError: ['The fields resource, permission, group must make a unique set.']
```